docs/source/developer/interactives.rst

```diff
@@ -183,9 +146,99 @@ Interactive Requirements
 
 A completed interactive requires the following to be included into the repository for release:
 
-- A ``README.md`` file explaining the interactive, linking to any necessary wiki pages.
+- A ``README.md`` file explaining the interactive, linking to any documentation or websites that future developers may find useful.
+  Uninteractives generally don't have a ``README.md`` file unless future developers may find it useful.
 - Extra libraries/scripts that have been used have been used and attributed correctly, see :ref:`licence-files`.
 - Be easily accessible on desktop and mobile, or show a disclaimer that it is suited for desktop use only.
 - Abides by the repository contribution style guidelines.
 - Must work in browsers updated within the last year.
   Therefore try and avoid experimental features but don't worry about supporting older browsers (but it's great if it can!).
+
+.. _adding-interactives:
+
+Adding Interactives
+==============================================================================
+
+Once you have developed your interactive (see :ref:`developing-interactives`), it's time to add it to a page.
+This includes embedding the interactive in the chapter text see :ref:`writing-guide-interactive`.
+
+.. note::
+
+  When adding an interactive using the Verto syntax, note that Verto uses the term ``name``, this is actually referring to the ``slug`` of the interactive.
+
+The interactive will also need to be added to the list of interactives in the interactive configuration file (see below).
+
+.. _interactive-configuration-files:
+
+Interactive Configuration Files
+------------------------------------------------------------------------------
+
+- **File Name:** ``interactives.yaml``
+
+- **Location:** ``interactives/content/structure/``
+
+- **Purpose:** Defines the list of interactives in the Field Guide.
+
+- **Required Fields:**
+
+  - ``interactives:`` A dictionary of all interactive data, where each key is a slug for an interacive. 
+
+  - The dictionary of interactive data must include:
+
+    - ``languages:`` A dictionary of all languages the interactive is available in, where each key is the locale.
+
+    - ``is_interactive:`` A boolean value indicating if the interactive can be interacted with.
+      This field was introduced so we could distingush between interactives and `uninteractives`_.
+      if the ``is_interactive`` field is set to ``false``, it means the user cannot interact with the content and it is not displayed in the interactives appendix.
```

Comment:
The line between interactive and uninteractive was blurred massively by the new persona_carousel uninteractive. That is indeed able to be interacted with

Reply:
(Just FYI), not sure how to or whether to talk about that in the docs